### DISCUSSION TITLE
Import button loading

### DISCUSSION THREAD

**Discussion by NaungYeHtet** [filamentphp/filament#11301] [Help]
### Package

Actions

### Package Version

v3.2

```
php artisan queue:batches-table
php artisan notifications:table
php artisan vendor:publish --tag=filament-actions-migrations
 
php artisan migrate

Actions\ImportAction::make()
    ->label(__('filament-actions::import.label', [
        'label' => ''
    ]))
    ->importer(MyImporter::class),
```

<img width="442" alt="image" src="https://github.com/filamentphp/filament/assets/49832780/968b20e5-4d69-4249-bb1c-b8e020651fd6">

Queue connection is database.
Enabled database notification in Panel.
Importing a csv file got loading forever. 
There is no error in laravel.log. 

I am using Sail, there is console and network error.

<img width="917" alt="image" src="https://github.com/filamentphp/filament/assets/49832780/4c98e6f7-4d73-4f9a-8e95-d6b1ef9aad8b">

<img width="978" alt="image" src="https://github.com/filamentphp/filament/assets/49832780/3069a6a6-5abc-4d90-9bfb-f8057bed43b1">

**Comment by leandrocfe** [comment]
Try to update Filament `composer update -W`
The current version is `3.2.27`

**Reply by NaungYeHtet** [reply]
Sorry for misunderstanding, my version was 3.2

**Reply by NaungYeHtet** [reply]
Still not working after composer update. It works once after composer update and there are a lot of retries on failed jobs table and queue:work terminal. After that the button got loading.
